README.md

```diff
@@ -328,3 +328,32 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## Generate ERD with django extensions
+```
+ pip install graphviz
+ pip install django-extensions
+ ```
+ Go to settings.py and add 
+ ```
+ 'django_extensions',
+ ```
+to INSTALLED_APPS
+
+Add the following code 
+```
+GRAPH_MODELS ={
+    'all_applications': True,
+    'graph_models': True,
+     }
+```
+ to settings.py:
+
```

Comment:
What if this line is written as:
Add the following code to `settings.py` file:
```
GRAPH_MODELS ={
    'all_applications': True,
    'graph_models': True,
     }
```
I think it's more readable this way.

Reply:
Sure. Will update it.